steely/plugins/cat.py

```diff
@@ -0,0 +1,15 @@
+'''cattos sure are lovely'''
+
+import requests
+import random
+
+__author__ = 'itsdoddsy'
+COMMAND = 'cat'
+FORMAT = ("png", "jpg")

```

Comment:
theoretically we could add `"gif"` to this but
- it would probably run up more data for those not on wifi
- i haven't tested it
- i'm too lazy to test it `👃`

Reply:
plus: gifs are buggy af